lib/models/tts.dart

```diff
@@ -277,6 +277,13 @@ class TtsModel extends ChangeNotifier {
     if (!enabled && !force) {
       return;
     }
+
+    if (model is StreamStateEventModel) {
```

Comment:
This should be done in the TTS isolate instead since the screen may not be open.

Reply:
Good catch, thought it could be a general tts function.